rts/Game/UI/Groups/GroupHandler.cpp

```diff
@@ -183,7 +184,13 @@ bool CGroupHandler::GroupCommand(int num, const std::string& cmd, bool& error)
 			if (group->units.empty())
 				return false;
 
-			camHandler->CameraTransition(0.5f);
+			float smoothness = 0.5f;
+			// check for optional camera smoothness argument
+			if (extraArgs.size() > 0) {
+				smoothness = StringToFloat(extraArgs[0].c_str());
```

Comment:
```suggestion
				smoothness = StringToFloat(extraArgs[0].c_str(), &error);
				if (error)
					return false;
```

Something like this maybe (my cpp skill are not great, this is just a suggestion).